src/config/authd-config.h

```diff
@@ -12,12 +12,13 @@
 #define AD_CONF_UNPARSED 3
 #define AD_CONF_UNDEFINED 2
 
-/**
- * @brief Structure that defines the force options for agent replacement.
```

Comment:
I think it's better if we keep this comment.

Reply:
Of course!
The rebase may have removed it